Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:13
+
+WORKDIR /Users/jim.ntare/Documents/work/devrepublic-bn-backend
```

Comment:
I believe this  working directory is only specific to you. Change this to sth like ```/usr/src/index``` or something accessible by everyone